indico/modules/events/papers/settings.py

```diff
@@ -54,6 +54,7 @@ def to_python(value):
     'announcement': '',
     'scale_lower': -3,
     'scale_upper': 3,
+    'hide_peer_reviewed_papers': False,
```

Comment:
How about just `hide_accepted`? *Anything* in these settings are related to paper peer reviewing ;)